RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -1685,7 +1702,8 @@ boolean acquireMP(int goal, int meat_reserve, boolean useFreeRests)
 	// TODO: move this to general effectiveness method
 	if(my_maxmp() - my_mp() > 300)
 	{
-		auto_sausageEatEmUp(1);
+		auto_sausageEatEmUp(1);		//this involve outfit changes which can lower our maxMP to below what goal was. which would cause infinite loop
+		goal = min(goal, my_maxmp());
```

Comment:
I think this might actually be redundant with the other change i made to catch infinite loops in the restore loop...
But I also think this is the only catcher that i actually tested in my recent inf loop spot